### PR TITLE
NuGet-publishing

### PR DESCRIPTION
**Pull request by HenrikWM**

### CODE REVIEW

File: src/AnonymousTokens/AnonymousTokens.csproj [hunk 1]
```diff
@@ -1,7 +1,24 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <Authors>Henrik Walker Moe, Martin Strand, Tjerand Silde</Authors>
```
Comment (martstr) [inline src/AnonymousTokens/AnonymousTokens.csproj:4]: Looks good. Is the permutation of names chosen randomly or based on some criterion?

(Background: In author listing in papers, some fields operate with sorting by contribution, some by alphabet, so someone could read meaning into it.)

Comment (HenrikWM) [reply src/AnonymousTokens/AnonymousTokens.csproj:4]: No just from the top of my head and my name popped up first as the default name :) Alphabetically seems like a logical choice. Is that by surname then? 

The authors are only visible on NuGet.org (and in Visual Studio when installing the package) when published: https://www.nuget.org/packages/Moq/ (to the right as a comma-separated string value)

@tjesi thoughts?

Comment (tjesi) [reply src/AnonymousTokens/AnonymousTokens.csproj:4]: I'm voting for alphabetically by surname.

File: src/AnonymousTokens/AnonymousTokens.csproj [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Authors>Henrik Walker Moe, Martin Strand, Tjerand Silde</Authors>
+    <Description>Privacy Pass and anonymous tokens implementation in .NET</Description>
```
Comment (tjesi) [inline src/AnonymousTokens/AnonymousTokens.csproj:5]: @martstr and I suggest `<Description>Anonymous tokens implementation in .NET based on Privacy Pass</Description>`

Comment (HenrikWM) [reply src/AnonymousTokens/AnonymousTokens.csproj]: Fixed and pushed

File: src/AnonymousTokens/AnonymousTokens.csproj (hunk 1 above)
Comment (HenrikWM) [reply src/AnonymousTokens/AnonymousTokens.csproj:4]: `<Authors>Henrik Walker Moe, Tjerand Silde, Martin Strand</Authors>`?

Comment (tjesi) [reply src/AnonymousTokens/AnonymousTokens.csproj:4]: Looks good 👍

Comment (HenrikWM) [reply src/AnonymousTokens/AnonymousTokens.csproj]: Pushed and also updated LICENSE

**Comment by HenrikWM** [comment]
Missing an XML-comment on `ECPointVerifier.PointIsValid()`. @martstr can you help me with a suggested text for this method?

**Pull request merged**